src/application.ts

```diff
@@ -20,6 +22,9 @@ export async function run(port: number, serveLocalFiles?: boolean, openPath?: st
         headers: {
           Accept: "application/vnd.mx.api.v1+json",
         },
+        httpsAgent: new https.Agent({
```

Comment:
Would it be better to offer this as a flag or configuration option instead of hard coding it?

Reply:
Good call, def should be behind a flag/config setting.

Reply:
I actually didn't put this behind a feature flag. Instead I'm checking if the host is for sand or qa, and setting this to `false` when it is.